components/Styling/Navigation.js

```diff
@@ -10,6 +10,13 @@ const StyledNavigationLink = styled(Link)`
   text-align: center;
 `;
 
+export const StyledCardLink = styled(Link)`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+`;
+
```

Comment:
As you do not use this component inside the `Navigation` component, I would recommend to put this one in a separate file.